etl/steps/data/garden/demography/2023-12-20/population_fariss.py

```diff
@@ -0,0 +1,84 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("population_fariss")
+    # Read table from meadow dataset.
+    tb = ds_meadow["population_fariss"].reset_index()
+    # Load Gleditsch dataset (country codes)
+    ds_gw = paths.load_dataset("gleditsch")
+    tb_gw = ds_gw["gleditsch_countries"].reset_index()
+
+    #
+    # Process data.
+    #
+    # Get code to country table
+    tb_gw = get_code_to_country(tb_gw)
+
+    # Keep only `latent_pop` indicator (dataset contains indicators from different sources)
+    tb = tb[tb["indicator"] == "latent_pop"]
+    # Keep relevant columns
+    tb = tb[["gwno", "year", "mean", "sd"]]
+
+    # Get country names
+    tb = tb.merge(tb_gw, left_on=["gwno"], right_on=["id"], how="left")
+    assert tb["country"].notna().all(), "Missing country names!"
+
+    # Restructure columns
+    tb = tb[["country", "year", "mean", "sd"]]
+
+    # Rename columns
+    tb = tb.rename(columns={"mean": "population", "sd": "population_sd"})
+
+    # Scale population (1e4)
```

Comment:
I wonder why this scale factor applies, it would be good to explain that in a comment.